packit/specfile.py

```diff
@@ -34,6 +34,18 @@ def __init__(self, path: Union[str, Path], sources_dir: Union[str, Path] = ""):
         self._patch_id_digits: Optional[int] = None
         self._uses_autosetup: Optional[bool] = None
 
+    def has_autochangelog(self):
+        """
+        The %autochangelog macro is in the specfile?
+
+        Returns:
+            bool: True if macro is in spec
+        """
+        for value in self.spec_content.section("%changelog"):
+            if "%autochangelog" == value.strip():
```

Comment:
FYI: [The walrus operator was added in Python 3.8](https://realpython.com/python-walrus-operator).
If we still want the packit code to be 3.6 compatible (I'm not sure now) then it's better to avoid it.